upgrader/lib/UpgradeSystemConfiguration.php

```diff
@@ -243,14 +248,18 @@ public function hasSetInstanceIdentifier() {
     }
 
     /**
-     * Set the instance identifier checksum
-     * @param $instanceIdentifierChecksum
+     * Set the instance identifier checksum     *
      */
-    public function setInstanceIdentifierChecksum($instanceIdentifierChecksum) {
-        $query = "INSERT INTO `hs_hr_config` (`key`, `value`) VALUES (?, ?)";
-        $dbConnection = $this->createDbConnection();
-        $statement = $dbConnection->prepare($query);
-        $statement->execute(array(self::KEY_INSTANCE_IDENTIFIER_CHECKSUM, $instanceIdentifierChecksum));
+    public function setInstanceIdentifierChecksum() {
+        $this->sysConf = $this->getSystemConfiguration();
```

Comment:
is that spacing right with other params?

Reply:
Yeah Samantha. This has 4 spaces for the indentation according to the PSR-2 code format